<launch>

  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen">
    <param name="image_width" type="int" value="640" />
    <param name="image_height" type="int" value="480" />
    <param name="framerate" type="int" value="30" />
    <param name="camera_frame_id" type="string" value="webcam" />
    <param name="video_device" type="string" value="/dev/video0" />
    <param name="autofocus" type="bool" value="0" />
    <!--- Needed for some devices --->
    <!-- <param name="pixel_format" value="yuyv" /> -->
  </node>
  
  <remap from="/image" to="/usb_cam/image_raw"/>
  
  <!-- comment this out when running on Odroid -->
  <node pkg="image_view" type="image_view" name="image_view" output="screen"/>

</launch>
